packages/console/src/notebook/ScriptEditor.tsx

```diff
@@ -138,7 +138,15 @@ class ScriptEditor extends Component<ScriptEditorProps, ScriptEditorState> {
     const wholeLineRange = range
       .setStartPosition(startLineNumber, startLineMinColumn)
       .setEndPosition(endLineNumber, endLineMaxColumn);
-    return model?.getValueInRange(wholeLineRange);
+
+    // Un-indent lines in case a user selected only lines within a block to run
+    const lines = model?.getValueInRange(wholeLineRange).split('\n');
+    const minIndent = Math.min(
```

Comment:
Using a `reduce` would probably be slightly more performant.

Reply:
Yes it would. Although probably not that important since it's run 1 time, but I can't pass up free performance!

[Link](https://codeburst.io/javascript-finding-minimum-and-maximum-values-in-an-array-of-objects-329c5c7e22a2)